.circleci/config.yml

```diff
@@ -715,19 +715,23 @@ commands:
             set -e
             image="<< parameters.image_name >>:<< parameters.tag >>"
             set -x
-            docker push "$image"
-            set +x
+            export NEW_POINT_RELEASE=true
             for tag in $(echo "<< parameters.extra_tags >>" | sed "s/,/ /g");
             do
-              set -x
               if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect << parameters.image_name >>:$tag >/dev/null 2>&1; then
                 echo "Image with Tag (<< parameters.image_name >>:$tag) already exists"
+                export NEW_POINT_RELEASE=false
```

Comment:
Shouldn't this also be in the config.yml.j2?

Reply:
Yup, missed that. I will actually create a new PR to add a  test to **test** that `config.yml` and `config.yml.j2` are in-sync

Reply:
Created: https://github.com/astronomer/ap-airflow/pull/80